docs/source/daml/intro/8_Exceptions.rst

```diff
@@ -0,0 +1,137 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+8 Exception Handling
+====================
+
+The default behavior in Daml is to abort the transaction on any error
+and rollback all changes that have happened until then. However, this
```

Comment:
I looked it up and it turns out English is weird: "rollback" is the noun form, and "roll back" is the verb form. (It's just like "login" vs "log in".)

```suggestion
and roll back all changes that have happened until then. However, this
```

Reply:
> I looked it up and it turns out English is weird:

who could have thought :slightly_smiling_face: